man/systemd-analyze.xml

```diff
@@ -1713,6 +1713,14 @@ LEGEND: M → sys_vendor (LENOVO) ┄ F → product_family (ThinkPad X1 Carbon G
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--drm-card=<replaceable>CARD</replaceable></option></term>
```

Comment:
i'd probably call this "device" rather than "card". At this time, "card" is a bit of an anachronism in many systems